release.nix

```diff
@@ -12,8 +12,8 @@ let
   config = {
     packageOverrides = pkgs:
       let
-        python_protobuf3_0 =
-          (pkgs.pythonPackages.protobufBuild pkgs.protobuf3_0).override {
+        python_protobuf3_1 =
+          pkgs.protobuf3_1.override {
```

Comment:
Did you mean `pkgs.python.protobuf` here?  The reason I ask is that you already have `pkgs.protobuf3_0` below in addition to this `pkgs.protobuf3_1` here.

Reply:
Was the old `pkgs.pythonPackages.protobufBuild` function used to convert `pkgs.protobuf` to a python module? 

Let me try removing this entire let clause.